python/cuml/utils/input_utils.py

```diff
@@ -19,11 +19,12 @@
 import cudf
 import cupy as cp
 import numpy as np
-import rmm
 import warnings
 
 from collections import namedtuple
 from collections.abc import Collection
+from copy import deepcopy as deepcp
```

Comment:
I'd rather just leave it as deepcopy to reduce confusion... it's not saving a lot of characters

Reply:
amusingly that causes a conflict with the parameter of the function `deepcopy`, so the rename is just to avoid that clash